libsolidity/ast/TypeProvider.cpp

```diff
@@ -347,6 +347,8 @@ TypePointer TypeProvider::forLiteral(Literal const& _literal)
 	case Token::FalseLiteral:
 		return boolean();
 	case Token::Number:
+		// TODO: return IntegerType, smallest fitting type in number of bits.
```

Comment:
Is this "TODO" correct? The correct type for a literal number should be "RationalNumberType".